entrypoint.sh

```diff
@@ -76,7 +76,7 @@ if [[ ! -f ${ROOTFS}/etc/systemd/system/kubelet.service ]]; then
 	Documentation=http://kubernetes.io/docs/
 
 	[Service]
-	Environment="KUBELET_VERSION=${K8S_VERSION}_coreos.0"
```

Comment:
Hmm, why is this changed?
What versions of CoreOS do you think we should support btw?
If things differ we can always do `coreos-alpha`, `coreos-stable` etc as separate OSes